extra/migrations/20170523000000_migrate_autosign_setting.rb

```diff
@@ -0,0 +1,29 @@
+require 'yaml'
+
+class MigrateAutosignSetting < ::Proxy::Migration
+  def migrate
+    puppetca_config = path(src_dir, "settings.d", "puppetca.yml")
+
+    unless File.exist?(puppetca_config)
+      return
+    end
+
+    to_migrate = YAML.load_file(puppetca_config)
+
+    output = migrate_puppetca_configuration(to_migrate)
+
+    write_yaml(puppetca_config, output)
+  end
```

Comment:
I can see two issues with this migration: it doesn't move unchanged config files into the destination directory (an example: https://github.com/theforeman/smart-proxy/blob/develop/extra/migrations/20161209000000_migrate_realm_settings.rb#L21), and it uses wrong directory to save the updated config file (```dst_dir``` points to the root. see https://github.com/theforeman/smart-proxy/blob/develop/extra/migrations/20161209000000_migrate_realm_settings.rb#L41 for an example).